### PR TITLE
[WIP][DEVOPS-1078] Deploy continuo em Dev

### PR DESCRIPTION
**Pull request by lucasfantacuci**

### CODE REVIEW

File: kubernetes/taskboard/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            docker.image('repo:5000/k8s-tools:v1').inside('-u 0') {
+                checkout scm
+                withCredentials([
+                        file(credentialsId: "DEVOPS_KUBECONFIG", variable: 'KUBECONFIG'),
+                ]) {
+                    sh 'cd ./kubernetes/taskboard && helm init --client-only'
+                    sh "cd ./kubernetes && ./upgrade-taskboard-dev.sh $VERSION"
+                }
+            }
+            slackSend (color: '#00FF00', message: "SUCCESSFUL: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})")
```
Comment (taksan) [inline kubernetes/taskboard/Jenkinsfile]: Message: "Build #${env.BUILD_NUMBER} successfully deployed to dev"

File: Jenkinsfile [hunk 2]
```diff
@@ -105,16 +100,21 @@ node("single-executor") {
...
-            def tag = "";
             if (params.RELEASE)
                 deployDocker("devopsobj/objective-taskboard:$projectVersion", "devopsobj", "")
             else {
-                tag = isMasterBranch() ? 'snapshot-master' : 'snapshot-'+projectVersion;
                 deployDocker("repo:5000/objective-taskboard:$tag", "repo-admin", "repo:5000")
             }
         }
+
+        stage ('Starting update taskboard') {
```
Comment (taksan) [inline Jenkinsfile]: Triggering taskboard-dev update

File: Jenkinsfile [hunk 3]
```diff
@@ -105,16 +100,21 @@ node("single-executor") {
...
+            if (!params.RELEASE)
```
Comment (taksan) [inline Jenkinsfile]: Why not update when a release is performed?

File: Jenkinsfile [hunk 4]
```diff
@@ -105,16 +100,21 @@ node("single-executor") {
             }
         }
 
+        def tag = sh(returnStdout: true, script: 'git rev-parse HEAD').trim()
```
Comment (taksan) [inline Jenkinsfile]: `env.GIT_COMMIT` - this variable already has the hash

File: Jenkinsfile [hunk 5]
```diff
@@ -105,16 +100,21 @@ node("single-executor") {
...
                 deployDocker("devopsobj/objective-taskboard:$projectVersion", "devopsobj", "")
             else {
-                tag = isMasterBranch() ? 'snapshot-master' : 'snapshot-'+projectVersion;
                 deployDocker("repo:5000/objective-taskboard:$tag", "repo-admin", "repo:5000")
             }
         }
+
+        stage ('Starting update taskboard') {
+            if (!params.RELEASE)
+                build job: 'sdlc/taskboard-continuos-deploy', parameters: [[$class: 'StringParameterValue', name: 'VERSION', value: tag]]
```
Comment (taksan) [inline Jenkinsfile]: `taskboard-continuos-deploy` -> `taskboard-continuous-deploy`

File: Jenkinsfile (hunk 3 above)
Comment (silvia-shimabuko) [reply Jenkinsfile]: Because of stage before is:

**Pull request merged**